app/constants/warnings.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  BATCH_REJECTED: 'uk.gov.defra.ffc.pay.warning.batch.rejected',
+  BATCH_QUARANTINED: 'uk.gov.defra.ffc.pay.warning.batch.quarantined',
+  PAYMENT_REJECTED: 'uk.gov.defra.ffc.pay.warning.payment.rejected', // PH
```

Comment:
We can probably remove these // PH and // DAX comments now - they were just to guide us as developing